spec/spec_helper.rb

```diff
@@ -1,5 +1,6 @@
 require 'bundler/setup'
 Bundler.setup
+require 'byebug'
```

Comment:
I thought about that, I'd say keeping it here because we could always use it without being bitten by it.